boards/adafruit/qt_py_esp32s3/adafruit_qt_py_esp32s3_procpu.dts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2023 Seeed Studio inc.
+ * Copyright (c) 2024 Ian Wakely
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32s3/esp32s3_wroom_n8.dtsi>
+#include <zephyr/dt-bindings/led/led.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include "adafruit_qt_py_esp32s3-pinctrl.dtsi"
+#include "seeed_xiao_connector.dtsi"
+#include <espressif/partitions_0x0_amp.dtsi>
+
+/ {
+	model = "Adafruit QT Py ESP32S3 PROCPU";
+	compatible = "seeed,xiao-esp32s3";
+
+	chosen {
+		zephyr,sram = &sram0;
```

Comment:
Please change to sram1 to be compliant with #80340

Reply:
addressed in second push.